src/Compatibility/Core/src/Android/Resources/layout/Toolbar.axml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.appcompat.widget.Toolbar
+<com.google.android.material.appbar.MaterialToolbar
```

Comment:
The `MaterialToolbar` lets you just modify/add/remove existing ToolBarItems which is how we recoded the ToolbarItem update code. 

If you leave this as the AppCompat ToolBar then it keeps duplicating items